ddl/ddl.go

```diff
@@ -76,6 +76,7 @@ var (
 	errInvalidWorker = terror.ClassDDL.New(codeInvalidWorker, "invalid worker")
 	// errNotOwner means we are not owner and can't handle DDL jobs.
 	errNotOwner              = terror.ClassDDL.New(codeNotOwner, "not Owner")
+	errCantDecodeIndex       = terror.ClassDDL.New(codeCantDecodeIndex, "can't decode index value, because %s")
```

Comment:
s/cant't/cannot

Reply:
Why can't use `can't`?

Reply:
I think `cannot` is more like formal writing.
I think `can’t` is best suited for informal writing. [F.Y.R](https://www.quickanddirtytips.com/education/grammar/cannot-can-not-or-can-t-what-s-the-difference)

Reply:
Done.